src/main/java/nextstep/subway/domain/command/FavoriteCommander.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.subway.domain.command;
+
+import lombok.RequiredArgsConstructor;
+import nextstep.subway.domain.entity.favorite.Favorite;
+import nextstep.subway.domain.query.PathReader;
+import nextstep.subway.domain.repository.FavoriteRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FavoriteCommander {
+    private final FavoriteRepository favoriteRepository;
+    private final PathReader pathReader;
+
+    public Long createFavorite(FavoriteCommand.CreateFavorite command) {
+        verifyPathExist(command.getSource(), command.getTarget());
+        Favorite favorite = Favorite.create(command);
+        favoriteRepository.save(favorite);
+        return favorite.getId();
+    }
+
+    public void deleteFavorite(FavoriteCommand.DeleteFavorite command) {
+        Favorite favorite = favoriteRepository.findByIdOrThrow(command.getFavoriteId());
+        favorite.verifyOwner(command.getMemberId());
+        favoriteRepository.deleteById(command.getFavoriteId());
```

Comment:
```suggestion
        favoriteRepository.deleteById(favorite.getFavoriteId());
```

command 보다는 검증 된 favorite 객체의 id를 이용하여 검증하는게 더 안정적이지 않을까요?

Reply:
넵!